sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -766,30 +768,39 @@ class ContainerAPITest extends APISpec {
         setupListBlobsTest(normalName, copyName, metadataName, uncommittedName)
 
         when:
-        Iterator<BlobItem> blobs = cu.listBlobsFlat(options, null).iterator()
+        Iterator<BlobItem> blobs = cc.listBlobsFlat(options, null).iterator()
 
         then:
         blobs.next().name() == normalName
         !blobs.hasNext() // Normal
     }
 
     // TODO (alzimmer): Turn this on once paged responses are available
-    /*def "List blobs flat options maxResults"() {
-        setup:
-        ListBlobsOptions options = new ListBlobsOptions().details(new BlobListDetails().copy(true)
-                .snapshots(true).uncommittedBlobs(true)).maxResults(2)
-        String normalName = "a" + generateBlobName()
-        String copyName = "c" + generateBlobName()
-        String metadataName = "m" + generateBlobName()
-        String uncommittedName = "u" + generateBlobName()
+    def "List blobs flat options maxResults"() {
+        setup:
+        def PAGE_SIZE = 2
+        def options = new ListBlobsOptions().details(new BlobListDetails().copy(true)
+            .snapshots(true).uncommittedBlobs(true)).maxResults(PAGE_SIZE)
+        def normalName = "a" + generateBlobName()
+        def copyName = "c" + generateBlobName()
+        def metadataName = "m" + generateBlobName()
+        def uncommittedName = "u" + generateBlobName()
         setupListBlobsTest(normalName, copyName, metadataName, uncommittedName)
 
-        when:
-        Iterator<BlobItem> blobs = cu.listBlobsFlat(options, null).iterator()
+        // TODO Turn this on once paged responses are available in sync client
+        /*when: "list blobs with sync client"
+        Iterator<BlobItem> blobs = cc.listBlobsFlat(options, null).iterator()
 
         then:
-        blobs.size() == 2
-    }*/
+        blobs.size() == PAGE_SIZE*/
```

Comment:
Commented out test

Reply:
Missed reactivating it. Will be in next commit.